queries.py

```diff
@@ -65,18 +112,20 @@ def print_result(result):
             .
         ?descriptor anno:start ?d_start
             ; anno:end ?d_end
+            ; anno:root/anno:pos ?pos
             .
+        FILTER (?pos NOT IN ("X", "PUNCT"))
         ?target anno:category "diseases"
             ; anno:start ?t_start
             ; anno:end ?t_end
             ; ^dc:relation ?rel
             .
+        ?rel rdf:label "malaria" .
```

Comment:
I forgot we were in `queries.py` - I was looking at the query in isolation. Makes sense.